templates/.rubocop.yml

```diff
@@ -16,7 +16,7 @@ Naming/PredicateName:
 
 Metrics/BlockLength:
   Exclude:
-  - config/initializers/simple_form_bootstrap.rb
+  - 'config/**/*'
```

Comment:
this does not really belong to this PR, right?

Reply:
my bad :joy: